include/oneapi/dpl/pstl/hetero/dpcpp/parallel_backend_sycl_radix_sort_one_wg.h

```diff
@@ -48,12 +48,15 @@ struct __subgroup_radix_sort
 
         using _KeyT = oneapi::dpl::__internal::__value_t<_RangeIn>;
         //check SLM size
-        if (__check_slm_size<_KeyT>(__q, __src.size()))
+        const auto __SLM_available = __check_slm_size<_KeyT>(__q, __src.size());
+        if (__SLM_available.first && __SLM_available.second)
             return __one_group_submitter<_SortKernelLoc>()(__q, ::std::forward<_RangeIn>(__src), __proj,
-                                                           std::true_type{} /*SLM*/);
-        else
+                                                           std::true_type{} /*SLM*/, std::true_type{} /*SLM*/);
```

Comment:
`std` -> `::std` in here and in a few more places in the next few lines of code.